pandapower/test/api/test_toolbox.py

```diff
@@ -165,6 +166,21 @@ def test_nets_equal():
     assert tb.nets_equal(original, net, atol=0.1)
     assert tb.nets_equal(net, original, atol=0.1)
 
+    # check controllers
+    original.trafo.tap_side.fillna("hv", inplace=True)
+    net1 = original.deepcopy()
+    net2 = original.deepcopy()
+    pp.control.ContinuousTapControl(net1, 0, 1.0)
+    pp.control.ContinuousTapControl(net2, 0, 1.0)
+    c1 = net1.controller.object.at[0]
+    c2 = net2.controller.object[0]
```

Comment:
write c1 = net1.controller.at[0, 'object'] and 
c2 = net2.controller.at[0, 'object'] instead